### PR TITLE
Allow admins to turn off HTML descriptions

### LINKED ISSUE #650: Long Descriptions with HTML truncates weird
TODO: See if it can be reproduced

What was seen:
"If a user puts HTML in the description of their notebook and it's a long description when the notebook is shown in a search result, the description gets cut short which can leave tags open. This then plays havoc with the rendering of the page and can result in parts of different notebooks being stiched together to appear as one, or the tags and stats for the notebook to just not be rendered."

### LINKED ISSUE #681: Add option to turn off HTML descriptions, pass all display of description/title through a sanitization function on output

### PR DESCRIPTION
**Pull request by mcrutch**
Alternative way to Close #650
Closes #681

### CODE REVIEW

File: app/views/application/_notebook_tile_small.slim [hunk 1]
```diff
@@ -16,7 +16,7 @@ div.info.square
         '  Notebook description:
       a.nounderline.notebook-description.tooltips href="#{notebook_path(notebook)}" title="#{notebook.description}"
         i.glyphicon.glyphicon-info-sign aria-hidden="true"
-        span.sr-only ==notebook.description
+        span.sr-only ==strip_tags(notebook.description)
```
Comment (mcrutch) [inline app/views/application/_notebook_tile_small.slim:19]: This is for screen readers only so wanted to strip tags but not "render" it to possibly break things

File: config/settings.yml [hunk 2]
```diff
@@ -151,6 +151,7 @@ video:
 
 # Notebook entries
 notebooks:
+  allow_html_description: false
```
Comment (mcrutch) [inline config/settings.yml:154]: defaulted to false because this is a MUCH safer mode of operation

**Comment by manfromjupyter** [comment]
@mcrutch, how do admins disable the HTML descriptions or is this only for system admins that can add the environment variable?

**Comment by mcrutch** [comment]
Only SA's not UI administrators

**Review by manfromjupyter — APPROVED** [review]
Code looks good

**Pull request merged**